### PR TITLE
add institutional partners csv, please populate!

### PR DESCRIPTION
**Pull request by CJ-Wright**
Hi everyone,

I'd like to start keeping track of our institutional partners. I'd like to define this fairly broadly to include entities that somehow contribute to the conda-forge ecosystem. 

These contributions could include (but are not limited to)
1. Storage for artifacts
2. Build infrastructure/time
3. Core Developer support
4. etc.

There are a couple reasons to keep track of this data:
1. It would be good to recognize our partners, without whom we might not be able to operate. Part of this recognition could be a listing on our website or in slides we present at conferences, universities, etc. An important piece of this would be the logo of the company. Please make certain that each logo is backed up with a written statement providing use of the logo for this purpose.
2. It would be good to know what kind of contributions we're accepting (build time, space, dev time, etc.) and from whom.

I've reformatted the file into yaml so we can have arbitrary keys. Please try to keep the keys self descriptive and please feel free to change the schema a little (for instance if you need two logo urls, please put them as a list, or if you need one as a main and others as alt please choose a data structure that reflects that).

@conda-forge/core please help populate this!

TODO:
- [x] Update initial post with context about why this thing exists, known applications of .csv
- [ ] If you're going to list your company, make sure that you've received written approval and then archive that somewhere. 
- [ ] Figure out where we're going to archive these written approvals on numfocus drive
- [ ] Decide on what keys we want to have in this csv
- [x] convert csv to yaml

### CODE REVIEW

**Comment by scopatz** [comment]
Numfocuse should be added

**Comment by scopatz** [comment]
Also should probably add Anaconda and Quansight

**Comment by ocefpaf** [comment]
@CJ-Wright can we add an extra field for "alternative website?"

I know it is an odd request but IOOS has two ( ioos.noaa.gov and ioos.us) and they would like to have both listed.

**Comment by SylvainCorlay** [comment]
> Also should probably add Anaconda and Quansight

QuantStack

**Comment by jakirkham** [comment]
I think it would help to clarify what constitutes an institutional partner. What things do they need to be doing to be listed here? Is there a threshold? Does ordering matter?

**Comment by jakirkham** [comment]
(Would also suggest adding NVIDIA, who has supplied some funding and developer time)

**Comment by ocefpaf** [comment]
> I think it would help to clarify what constitutes an institutional partner. What things do they need to be doing to be listed here? Is there a threshold? Does ordering matter?

I don't care about ordering, maybe we can do alphabetical. But yes, it would be nice what each partner is bringing to the table. I guess that most of them is implied in the "Associated Core Developers (if any)" but others, like Anaconda and NVIDIA, bring actual $$$ to the table.

**Comment by ocefpaf** [comment]
> (Would also suggest adding NVIDIA, who has supplied some funding and developer time)

I believe that @CJ-Wright wants for us to edit it and add stuff, right @CJ-Wright? If so I guess you are the most qualified person to add the NVIDIA line.

**Comment by CJ-Wright** [comment]
Yes please add and edit!

**Comment by jakirkham** [comment]
Are we able to add 2 logo images? Or should those just be 2 different entries?

**Comment by ocefpaf** [comment]
> Are we able to add 2 logo images? Or should those just be 2 different entries?

Similar to my question above, if we are going to use this in any automated tool later, maybe have a "field", "alt field" is better than "field-a, field-b", no?

If so I'm OK adding logo, alt logo, url, alt url.

**Comment by jakirkham** [comment]
Yeah not sure how we want to do it. In our case it sounds like we want the NVIDIA and the RAPIDS logos. So it's not an alt per-se. Can just add 2 entries like you did if that makes the most sense.

**Comment by ocefpaf** [comment]
> Yeah not sure how we want to do it. In our case it sounds like we want the NVIDIA and the RAPIDS logos. So it's not an alt per-se. Can just add 2 entries like you did if that makes the most sense.

Yep. That is different from my case them. I would do two lines then.

**Comment by SylvainCorlay** [comment]
> I think it would help to clarify what constitutes an institutional partner. What things do they need to be doing to be listed here? Is there a threshold? Does ordering matter?

Jupyter distinguishes sponsors (who bring ¥€$), and Institutional Partners which are organizations that employ steering committee members. https://jupyter.org/about

File: src/inst_partners.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  Logo URL: https://www.worksonarm.com/wp-content/themes/worksonarm2019/assets/images/logo.png
+  Site URL: https://www.worksonarm.com/
+  contribution type: infrastructure
+IOOS:
+  Description: The U.S. Integrated Ocean Observing System
+  Developers: ocefpaf
+  Logo URL: https://cdn.ioos.noaa.gov/media/2017/12/IOOS_logo.png
+  Site URL:
+    - https://ioos.noaa.gov/
+    - https://ioos.us/
```
Comment (ocefpaf) [inline src/inst_partners.yaml:21]: @CJ-Wright I hope this is fine.

File: src/inst_partners.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  Site URL: https://osuosl.org
+  contribution type: infrastructure
+Works on Arm:
+  Description: Works on Arm project provides aarch64 machines for long running jobs
+  Logo URL: https://www.worksonarm.com/wp-content/themes/worksonarm2019/assets/images/logo.png
+  Site URL: https://www.worksonarm.com/
+  contribution type: infrastructure
+IOOS:
+  Description: The U.S. Integrated Ocean Observing System
+  Developers: ocefpaf
```
Comment (ocefpaf) [inline src/inst_partners.yaml:17]: I followed @jjhelmus' lead here. I guess we can get more info on people with the GH handle than just the name ;-p

File: src/inst_partners.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  Logo UR: https://osuosl.org/images/osllogo-print_0.png
+  Site URL: https://osuosl.org
+  contribution type: infrastructure
+Works on Arm:
+  Description: Works on Arm project provides aarch64 machines for long running jobs
+  Logo URL: https://www.worksonarm.com/wp-content/themes/worksonarm2019/assets/images/logo.png
+  Site URL: https://www.worksonarm.com/
+  contribution type: infrastructure
+IOOS:
+  Description: The U.S. Integrated Ocean Observing System
```
Comment (isuruf) [inline src/inst_partners.yaml:16]: Other descriptions are about the contribution.

**Pull request merged**